linode/sshkeys/framework_datasource.go

```diff
@@ -59,6 +63,8 @@ func listSSHKeys(
 	client *linodego.Client,
 	filter string,
 ) ([]any, error) {
+	tflog.Trace(ctx, "client.ListSSHKeys(...)")
```

Comment:
We probably can log the `filter` as well? Just to show more information.

Reply:
Added to ctx in Read function above